package errors

import "errors"


var ErrIterationsExhausted = errors.New("iterations exhausted, and a result could not be found")
